test/verify/check-system-info

```diff
@@ -421,6 +425,65 @@ class TestSystemInfo(MachineCase):
 
         self.machine.execute("umount /sys/class/dmi")
 
+    def testCPUSecurityMitigations(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go('/system/hwinfo')
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(No mitigations available)')
+
+        m.write('/usr/local/bin/lscpu', lscpu)
+        m.execute('chmod +x /usr/local/bin/lscpu')
+        b.reload(ignore_cache=True)
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(nosmt)')
+        b.wait_in_text('#cpu-mitigations-dialog tr:contains(nosmt) .btn.active', 'Off')
+
+        m.write('/etc/cmdline', 'nosmt')
+        m.execute('mount --bind /etc/cmdline /proc/cmdline')
+        self.assertIn('nosmt', m.execute('cat /proc/cmdline'))
+        b.reload(ignore_cache=True)
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(nosmt)')
+        b.wait_in_text('#cpu-mitigations-dialog tr:contains(nosmt) .btn.active', 'On')
+
+        m.write('/etc/cmdline', 'nosmt=force')
+        b.reload(ignore_cache=True)
+        b.enter_page('/system/hwinfo')
+        b.wait_present('#hwinfo a:contains(Mitigations)')
+        b.click('#hwinfo a:contains(Mitigations)')
+        b.wait_present('#cpu-mitigations-dialog tr:contains(No mitigations available)')
+        m.execute('umount /proc/cmdline')
```

Comment:
Let's also test an unknown value (if the code starts to recognize the "force" as a known value)